epilogue-processor/src/main/java/edu/wpi/first/epilogue/processor/ElementHandler.java

```diff
@@ -61,14 +62,24 @@ protected TypeMirror dataType(Element element) {
    * @return the name specified in the {@link Logged @Logged} annotation on the element, if present;
    *     otherwise, the field or method's name with no modifications
    */
-  public String loggedName(Element element) {
+  public static String loggedName(Element element) {
     var elementName = element.getSimpleName().toString();
     var config = element.getAnnotation(Logged.class);
 
     if (config != null && !config.name().isBlank()) {
       return config.name();
     } else {
-      return elementName;
+      // Delete common field prefixes (k_Name, m_name, s_name)
+      elementName = elementName.replaceFirst("^[msk]_", "");
```

Comment:
`_` is also sometimes used as a prefix for private members

Reply:
Good callout